Runtime/Rule/ListContainsRule.cs

```diff
@@ -13,15 +14,16 @@ public class ListContainsRule : MonoBehaviour, IRule
         /// <summary>
         /// The objects to check against.
         /// </summary>
-        [DocumentedByXml]
-        public List<Object> objects = new List<Object>();
+        [Serialized]
+        [field: DocumentedByXml]
+        public UnityObjectObservableList Objects { get; set; }
 
         /// <inheritdoc />
         [RequiresBehaviourState]
         public virtual bool Accepts(object target)
         {
```

Comment:
```suggestion
         {
             if (Objects == null)
             {
                 return false;
             }
```
This saves the cast and null-check for `targetObject` below.